.travis.yml

```diff
@@ -2,18 +2,39 @@ sudo: false
 language: php
 php:
   - 7.2
-
-branches:
-  only: 
-    - master
+services:
+  - docker
+addons:
+  apt:
+    packages:
+      - docker-ce
+env:
+  - DOCKER_COMPOSE_VERSION=1.23.2
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - "$HOME/.npm"
```

Comment:
Docker layer caching should be added as well if possible, see https://github.com/versionpress/versionpress/pull/1415#issuecomment-476875059.

Reply:
I implemented this approach: https://github.com/travis-ci/travis-ci/issues/5358#issuecomment-248915326

Reply:
The latest build is still slow but that might be because the cache was only being created? Let's see in the newer builds (I saw you just triggered one...).

![Screenshot 2019-03-27 at 16 03 30](https://user-images.githubusercontent.com/101152/55087254-1ba56f80-50aa-11e9-9771-5515ea4967ba.png)

Reply:
The build is now failing, marking is as not resolved.

Reply:
How about we merge this now (commit bb928) and work on the docker caching in a separate PR as an improvement?

Reply:
Agree, I'd also do it that way. Can you please update this PR so that the build passes?

Reply:
Done!

Reply:
Thanks. Just remember, please no force-pushing 😄.